src/DataTable/README.md

```diff
@@ -1062,7 +1062,7 @@ a responsive grid of cards.
 ### Customizing number of Cards shown per row
 Use `columnSizes` prop of `CardView` component to define how many `Cards` are shown per row at each breakpoint.
 
-`columnSizes` is an object containing the desired column size at each breakpoint. The example below shows 1 `Card` per row at `xs` breakpoint, 2 `Cards` at `sm` and `md`, and 4 `Cards` at `lg` and higher. You can read more about the API at https://react-bootstrap.netlify.app/layout/grid/.
+`columnSizes` is an object containing the desired column size at each breakpoint. The example below shows 1 `Card` per row at `xs` breakpoint, 2 `Cards` at `sm` and `md`, and 4 `Cards` at `lg` and higher. You can read more about the API at [React-Bootstrap additional documentation](https://react-bootstrap-v4.netlify.app/layout/grid/).
```

Comment:
```suggestion
`columnSizes` is an object containing the desired column size at each breakpoint. The example below shows 1 `Card` per row at `xs` breakpoint, 2 `Cards` at `sm` and `md`, and 4 `Cards` at `lg` and higher. You can read more about the API at [React-Bootstrap's grid documentation](https://react-bootstrap-v4.netlify.app/layout/grid/).
```

I think this wording might be a bit better. And I'm pretty sure you need to add `'s` anyway to be grammatically correct.

Reply:
I agree on `additional` -> `grid`. But I would not add `'s` here from my experience. Also this is answer from ChatGPT:
```
React-Bootstrap grid documentation: This implies a focus on the documentation itself. It's like saying "the documentation for the React-Bootstrap grid."
React-Bootstrap's grid documentation: This implies possession. It's like saying "the grid documentation that belongs to React-Bootstrap."

Reply:
Thanks, corrected

Reply:
> React-Bootstrap's grid documentation

Approved by @adamstankiewicz